pkg/providerConfigs/adjustDriver.go

```diff
@@ -143,6 +145,16 @@ func AdjustGitopsRepo(
 		removeAllWithLogger(strings.ToLower(fmt.Sprintf("%s/%s-%s/templates/workload-vcluster/40-cloudflare-origin-issuer-crd.yaml", gitopsRepoDir, cloudProvider, gitProvider)))
 		removeAllWithLogger(strings.ToLower(fmt.Sprintf("%s/%s-%s/templates/workload-vcluster/41-cloudflare-origin-ca-issuer.yaml", gitopsRepoDir, cloudProvider, gitProvider)))
 		removeAllWithLogger(strings.ToLower(fmt.Sprintf("%s/%s-%s/templates/workload-vcluster/45-cloudflare-origin-issuer.yaml", gitopsRepoDir, cloudProvider, gitProvider)))
+
+		for _, cloud := range aiSupport {
+			if cloud == cloudProvider {
+				removeAllWithLogger(strings.ToLower(fmt.Sprintf("%s/%s-%s/templates/ai-cluster/cloudflare-origin-issuer", gitopsRepoDir, cloudProvider, gitProvider)))
```

Comment:
Don't have to fix it right now but it's a red herring to create paths by concatenating a string.

It's better to use `path.Join()` or `filepath`. Let's try to slowly move in that direction.

Reply:
done :)